muikku/src/main/webapp/resources/scripts/notificationqueue/notificationqueue.js

```diff
@@ -52,6 +52,10 @@
       }
     },
     
+    remove: function (item) {
+      $(item).hide(this.options.hide.effect, this.options.hide.options);
```

Comment:
Refactor _onRemoveClick to use this method so the functionality is not implemented twice on this widget